mindmap/static/js/src/mindmapEdit.js

```diff
@@ -0,0 +1,17 @@
+/* Javascript for MindMapXBlock. */
+function MindMapXBlock(runtime, element) {
+
+    $(element).find('.save-button').bind('click', function() {
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit');
+        var data = {
+            display_name: $(element).find('input[name=mindmap_display_name]').val(),
+        };
+        $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
+          window.location.reload(false);
+        });
```

Comment:
Please add a function in case that this fails. It would be something like this:
```
 $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
          window.location.reload(false);
   }).fail(function() { console.log('something went wrong');});
```

Check [this](https://stackoverflow.com/a/2833968) if you want to handle it

Reply:
Done